django_slack_oauth/models.py

```diff
@@ -5,21 +5,17 @@
 
 from jsonfield import JSONField
 
-__all__ = (
-    'SlackUser',
-)
 
+class SlackOAuthRequest(models.Model):
```

Comment:
What about backward compatibility ? 
Renaming just removes the old table. Not a great situation for lib users.